docs/DEPLOYMENT.md

```diff
@@ -27,6 +27,13 @@ cd api
 
 Navigate to the `config/00-config/` and edit `31-api-secret.yaml` . Set GitHub `oauth` client id and client secret.
 
+To create a GitHub OAuth follow the steps given [here][oauth-steps]. For now add any URL in place for `Homepage URL` and `Authorization callback URL` for eg. `http://localhost:5000`. We will update this URL once we deploy the UI pod.
```

Comment:
Could you please create links for all the headings as we have in the development docs ?